EIPS/eip-X.md

```diff
@@ -0,0 +1,201 @@
+Ethereum blocks are bursting at the seams due to increasingly high demand for decentralized applications. Ever since the first serious spikes in adoption in 2017 (cryptokitties), the Ethereum community has consistently and vocally demanded scaling solutions.
+
+Since day 0 of Ethereum, the investigation and expectation in scaling solutions is two-fold -- scale from both Layer 1 upgrades as well as Layer 2 adoption. 
+
+For details regarding specification design rationale and decisions, see the following:
+* [Serenity Design Rationale](https://notes.ethereum.org/@vbuterin/rkhCgQteN)
+* [Ben Edgington's Eth2 Annotated Spec](https://benjaminion.xyz/eth2-annotated-spec/)
+* [Vitalik Buterin's Eth2 Annotated Spec](https://github.com/ethereum/annotated-spec/blob/master/phase0/beacon-chain.md)
+
+## Backwards Compatibility
+<!--All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
+This EIP does not introduce any layer 1 changes to Ethereum mainnet and is thus entirely forwards/backwards compatible.
+
+Instead, this EIP lays the groundwork for future backwards incompatibilities through the introduction of the new eth2 consensus mechanism in which Ethereum will be integrated in subsequent phases. To secure this mechanism, users move ether into the beacon chain and additional ether is issued. This EIP is a commitment to this path as being canonical and directly informing the future and roadmap of Ethereum mainnet.
+
+## Implementation
+<!--The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.-->
+
+The following implementations of eth2 Phase 0 exist and at the time of writing at least five are near production-ready and in testnet/audit phase:
+
+* [Cortex](https://github.com/NethermindEth/cortex) (.net)
+* [Lighthouse](https://github.com/NethermindEth/cortex) (rust)
+* [Lodestar](https://github.com/ChainSafe/lodestar) (javascript)
+* [Nimbus](https://github.com/status-im/nim-beacon-chain) (nim)
+* [Prysm](https://github.com/prysmaticlabs/prysm/) (go)
+* [Teku](https://github.com/pegasyseng/teku) (java)
+* [Trinity](https://github.com/ethereum/trinity/) (python)
+
+In addition to implementations of the spec, the deposit contract is [implemented in Solidity](), along with a [formal verification]() of the bytecode.
```

Comment:
Missing links, blocked by new Eth2 spec release.
- Solidity code (dev branch): https://github.com/ethereum/eth2.0-specs/tree/dev/solidity_deposit_contract
- Formal verification. Report was published, but we may want to tag a release that links it.

```suggestion
In addition to implementations of the spec, the deposit contract is [implemented in Solidity](), along with a [formal verification](https://github.com/runtimeverification/verified-smart-contracts/blob/master/deposit/deposit-formal-verification.pdf) of the bytecode.
```